### PR TITLE
knative-client v0.18.4 release CI setup

### PR DESCRIPTION
**Pull request by navidshaikh**
with a new test-image grpc-ping mirror setting at quay
/assign @dsimansk

### CODE REVIEW

**Comment by dsimansk** [comment]
/lgtm
/hold
@navidshaikh adding hold beucase release branch is missing, feel free to unhold once you push it.

File: ci-operator/config/openshift/knative-client/openshift-knative-client-release-v0.18.4.yaml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+      - destination_dir: .
+        source_path: /go/bin/kn
+  to: knative-client
+- dockerfile_path: openshift/ci-operator/knative-images/client/Dockerfile.cliartifacts
+  inputs:
+    bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/src/github.com/knative/client/kn-linux-amd64
+      - destination_dir: .
```
Comment (dsimansk) [inline ci-operator/config/openshift/knative-client/openshift-knative-client-release-v0.18.4.yaml:28]: Should we include s390x already?

Comment (navidshaikh) [reply ci-operator/config/openshift/knative-client/openshift-knative-client-release-v0.18.4.yaml:28]: lets do in next iteration with ppc64 and arm

**Pull request merged**